src/labels/CustomLabels.labels

```diff
@@ -1847,71 +1847,71 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <fullName>conMergeFoundContacts</fullName>
         <categories>Contact Merge</categories>
         <language>en_US</language>
-        <protected>false</protected>
+        <protected>true</protected>
         <shortDescription>conMergeFoundContacts</shortDescription>
         <value>Found Contacts</value>
     </labels>
     <labels>
         <fullName>conMergePageTitle</fullName>
         <categories>Contact Merge</categories>
         <language>en_US</language>
-        <protected>false</protected>
+        <protected>true</protected>
         <shortDescription>conMergePageTitle</shortDescription>
         <value>Contact Merge</value>
     </labels>
     <labels>
         <fullName>conMergePageTitleDetail</fullName>
         <categories>Contact Merge</categories>
         <language>en_US</language>
-        <protected>false</protected>
+        <protected>true</protected>
         <shortDescription>conMergePageTitleDetail</shortDescription>
         <value>Merge Duplicate Contact Records</value>
     </labels>
     <labels>
         <fullName>conMergeSearchPlaceholder</fullName>
         <categories>Contact Merge</categories>
         <language>en_US</language>
-        <protected>false</protected>
+        <protected>true</protected>
         <shortDescription>conMergeSearchPlaceholder</shortDescription>
         <value>Search Contacts</value>
     </labels>
     <labels>
         <fullName>conMergeSelectAll</fullName>
         <categories>Contact Merge</categories>
         <language>en_US</language>
-        <protected>false</protected>
+        <protected>true</protected>
         <shortDescription>conMergeSelectAll</shortDescription>
-        <value>select All</value>
```

Comment:
@jjbennett I had left this lower case because the original text was lower case, though I think it's probably better this way.